feature: add absolut imports

uriielfl opened the pull request:
**Descrição:**

Adicionar imports absolutos com o intuito de facilitar o desenvolvimento e compreensão do código, por exemplo:

_Relativo:_
`import { ABCD } from '../../components/ABCD';`

_Absoluto:_
`import { ABCD } from '@/components/ABCD';`

uriielfl committed:
feature: add absolut imports

ghost commented:
@uriielfl poderia adicionar o refactoring para o modelo de import absoluto no código já escrito tb?

uriielfl committed:
refactor: add absolut imports to existing code

uriielfl committed:
feature: add main components file

uriielfl committed:
fix: change css import order

uriielfl committed:
add: changeset

uriielfl committed:
fix: change imports in pages

change imports in page and add eslint rule to avoid the needing of
file extension in imports.

uriielfl commented:
https://github.com/all-contributors please add @uriielfl for absolut imports config implementation

uriielfl commented:
@all-contributors please add @uriielfl for absolut imports config implementation

uriielfl commented:
@all-contributors please add @uriielfl for code

guilhermeSDB committed:
Merge branch 'main' of https://github.com/Uno-Online/uno-frontend into feature/absolut-imports

guilhermeSDB commented:
@fell-lucas Pode me ajudar aqui? Prescisamos adicionar o yarn.lock ao .gitignore

fell-lucas committed:
chore: 🔧 update yarn.lock
